src/main/java/app/gpx_animator/core/renderer/plugins/InformationPlugin.java

```diff
@@ -85,12 +88,30 @@ public void renderFrame(final int frame, @Nullable final Point2D marker, @NonNul
         final var latLongString = getLatLonString(marker);
         final var speedString = getSpeedString(marker, time, frame);
 
-        final var text = information
-                .replace("%SPEED%", speedString)
-                .replace("%LATLON%", latLongString)
-                .replace("%DATETIME%", dateTimeString);
-
+        final var gpsTime = getTime(marker);    //TODO --Get strings from resource
+        var gpsDateTimeString = "Unknown";
+        var gpsDiffTime = 0L;
+        var gpsDiffTimeString = "";
+        var gpsStatus = false;
+        var gpsLostTimeString = "";
+        var gpsStatusString = "NOTIME";           //TODO --Get strings from resource
+        if (gpsTime > 0) {
+            gpsDiffTime = time - gpsTime;
+            gpsDateTimeString = dateFormat.format(gpsTime);
+            gpsDiffTimeString = DurationFormatUtils.formatDuration(gpsDiffTime, "+H:mm:ss");
+            gpsStatus = gpsTimeout <= 0 || gpsDiffTime < gpsTimeout;
+            gpsLostTimeString = gpsStatus ? "" : gpsDiffTimeString;
+            gpsStatusString = gpsStatus ? "OK" : "LOST";      //TODO --Get strings from resource
```

Comment:
This `TODO` should be resolved, it's an easy one.

Reply:
Not easy, bacuse the plugin don't know the language resoure object.